### DISCUSSION TITLE
Travel from Seattle to FHL

### DISCUSSION THREAD

**Discussion by sr320** [sr320/ceabigr#7] [General]
Lets use this discussion to share information on best ways to get to FHL including carpooling etc. 

Here is [ferry info](https://wsdot.com/ferries/schedule/scheduledetail.aspx?departingterm=1&arrivingterm=10&roundtrip=true).

**Comment by yaaminiv** [comment]
Interested in carpooling from Seattle to FHL! @sr320 is there a specific ferry we're trying to take?

**Reply by sr320** [reply]
Seems like east coast flights might dictate (unless folks are coming in day(s) early.

Is 1:50 even an option? When are folks arriving?

<img width="532" alt="Screen Shot on 2022-02-02 at 10-28-55" src="https://user-images.githubusercontent.com/4442039/152215759-30d9bfd6-82a6-4c14-b30d-c5d0128d6e0b.png">

**Reply by yaaminiv** [reply]
I'll be in town a few days before so I am flexible on 2/20 ferry.

**Reply by shellywanamaker** [reply]
I'm arriving from BOS to SEA on Feb 20 at **10:44 AM** on Delta flight DL341 and departing from SEA to BOS on Feb 23 at **11:10 PM** on Alaska flight AS326

**Reply by jarcasariego** [reply]
My flight lands at 11:50 AM (Alaska 225) the 20th. My return is on the 23rd at 10 PM

**Reply by laurahspencer** [reply]
I’m also interested in carpooling, as I won’t be able to take my car up to FHL.

**Reply by sr320** [reply]
I will pursue trying to get a van from UW, but that will not hold everyone. We can also take my truck, but I would need someone to drive it.

**Reply by jarcasariego** [reply]
I can drive if needed

**Reply by ksil91** [reply]
I'm flexible on the Feb. 20 ferry as I'm arriving on the 19. 
Arriving from CHS to SEA on Feb. 19 at 8:30p on Alaska flight 787, and departing 10pm on Feb. 23 on American flight 2793.

**Reply by laurahspencer** [reply]
UPDATE: I actually have to be back to Seattle by Wednesday morning, so will probably grab a ferry back Tuesday night.  I'll drive my truck up separately. Depending on the ferry reservation situation I'll either park it at Anacortes and catch a ride to FHL with you all, or just drive it onto the ferry. 

Are you all catching the 3:20pm ferry from Anacortes on Sunday?

**Comment by sr320** [comment]
>Are you all catching the 3:20pm ferry from Anacortes on Sunday?

Ferry - I think @AHuffmyer has reservation on 3:20. I have one on the 5:30 just because yesterday there were no reservation slots available for 3:20 (will likely try to catch)

**Comment by AHuffmyer** [comment]
I have a reservation for the 1:25 ferry on Sunday from Anacortes to Friday Harbor!

**Comment by ksil91** [comment]
Is there a carpool plan yet for those of us without cars to get to the ferry? I'm flexible on ferry time (would prefer the 3:20p but can do anytime). I can also (probably?) ride with Laura to the ferry, but would need a ride back to Seattle on Wed.

**Reply by sr320** [reply]
@ksil91 I am responsible for you- will have a mini van and leaving airport probably around 12:30. Picking up van at UW Sunday AM.

**Reply by yaaminiv** [reply]
Should @ksil91 and I meet up with you at SAFS to be picked up on your way north from the airport?

**Reply by sr320** [reply]
I think now @ksil91 is riding up with @laurahspencer

**Reply by sr320** [reply]
But yes that sounds good..
